pkg/reconciler/instances/ory/action_test.go

```diff
@@ -517,7 +491,87 @@ func TestOryDbSecret(t *testing.T) {
 		})
 	}
 }
+func TestOryDbSecret_IsUpdate(t *testing.T) {
+	t.Run("should return false when applying the same secret", func(t *testing.T) {
+		// given
+		name := types.NamespacedName{Name: "test-db-secret", Namespace: "test"}
+		ctx := context.Background()
+		k8sClient := fake.NewSimpleClientset()
+		logger := zaptest.NewLogger(t).Sugar()
```

Comment:
You can freely move this logger initialisation above all the test cases and reuse it.